src/servers/sapphire_zone/Zone/House.cpp

```diff
@@ -0,0 +1,68 @@
+#include <set>
+#include <string.h>
+
+#include <Logging/Logger.h>
+#include <Exd/ExdDataGenerated.h>
+
+#include "House.h"
+
+#include <unordered_map>
+#include "Framework.h"
+
+extern Core::Framework g_fw;
+
+Core::House::House( uint32_t houseId, uint32_t landSetId, uint8_t landId, uint8_t wardNum, uint16_t zoneId ) :
+  m_houseId( houseId ),
+  m_landSetId( landSetId ),
+  m_landId( landId ),
+  m_wardNum( wardNum ),
+  m_zoneId( zoneId )
+{
+  memset( &m_houseParts, 0x00, 8 );
+  memset( &m_HouseMsg, 0x00, 193 );
```

Comment:
name should reflect what it is in game welcomeMsg or something like that, i don't recall what exactly it was...

Reply:
Also always lowercase after m_